app/views/applications/show.html.erb

```diff
@@ -0,0 +1,62 @@
+<br>
+<h1>Name of Applicant: <%= @application.name %></h1>
+
+<section id="applicant-info">
+<p>
+  Applicant Address:
+  <%= @application.street_address %>,
+  <%= @application.city %>, <%= @application.state %>, <%= @application.zip_code %>
+</p>
+<p>Description of why the applicant says they'd be a good home for this pet(s): <%= @application.description %></p>
+</section>
+
+<br>
+
+<section id="pets-applied-for">
+<h2>Pets Applied for:</h2>
+<ul>
+  <% @application.pets.each do |pet| %>
+    <li><%= link_to pet.name, "/pets/#{pet.id}" %></li> 
+  <% end %>
+</ul>
+</section>
+
+<section id="app-status">
+<p> Application's status: <%= @application.status %></p>
+</section>
+<br>
+
+<section id="application-submit">
+  <% if @application.pets.count != 0 %>
+    <%= form_with url: "/applications/#{@application.id}", method: :patch, data: { turbo: false } do |form| %>
+      <br><%= form.label :adopt_description, "Why would you make a good owner for these pet(s)?" %>
+      <%= form.text_field :adopt_description %>
+      <%= form.submit "Submit Application" %>
+    <% end %>
+  <% end %>
+</section>
+<br>
+
+<section id="pet-search">
+<% if @application.status == "In Progress" %>
+  <h3>Add a Pet to this Application</h3>
+  <%= form_with url: "/applications/#{@application.id}", method: :get, data: { turbo: false } do |form| %>
+    <%= form.hidden_field :search, value: "pet" %>
```

Comment:
you could probably get rid of this hidden field, and just check in the controller action for the presence of `pet_name` in the params object